### PR TITLE
CKHZ|CKID Equipment Change

### PR DESCRIPTION
**Pull request by Thomas-Benson**
Replaced Kinemetrics Q330/3 (2564) with Nanometrics Centaur (9891)
For LC 10 (EH-ZNE)
@mnaguit do I need to update install/stream.csv for this one, or does it remain the same?

### CODE REVIEW

File: install/dataloggers.csv [hunk 1]
```diff
@@ -978,6 +978,7 @@ Nanometrics,Centaur CTR4-6S,9876,Wakanui,,2023-05-17T21:39:00Z,9999-01-01T00:00:
 Nanometrics,Centaur CTR4-6S,9883,Mangawhero Road,,2023-04-18T03:01:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,9887,Logan Vale Farm,,2023-01-18T04:14:00Z,9999-01-01T00:00:00Z
 Nanometrics,Centaur CTR4-6S,9888,Te Maipa,,2023-04-18T02:55:00Z,9999-01-01T00:00:00Z
+Nanometrics,Centaur CTR4-6S,9891,Cape Kidnappers,,2023-05-04T02:09:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline install/dataloggers.csv]: please modify the opening for Centaur: `2023-05-25/04:20UTC`

**Comment by mnaguit** [comment]
Hi @Thomas-Benson . No changes in streams. All good for `LC 10: EH-ENZ` (non-axial) streams. 
We have to add the firmware version `4.5.20` for Centaur SN `9891` (in `install/firmware.csv`).   
Thanks!

**Comment by Thomas-Benson** [comment]
Ok, I'll update that. Something wnet wrong with the ckecks though, and I'm not sure how to fix that.

**Comment by mnaguit** [comment]
For the Travis error, the start date of Centaur has to be modified as mentioned in my previous comment (in `install/dataloggers.csv`).
There is a conflict as Centaur opens before the old logger (Q330/3) was closed.
@Thomas-Benson

**Review by mnaguit — APPROVED** [review]
Yay! Looks good! 👍

**Pull request merged**